$MirOS: ports/audio/libvorbis/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:42:58 tg Exp $
$OpenBSD: patch-configure,v 1.6 2003/12/14 00:52:22 naddy Exp $
--- configure.in.orig	Tue Jun 21 15:47:23 2005
+++ configure.in	Wed Nov  9 19:20:17 2005
@@ -94,7 +94,7 @@ if test -z "$GCC"; then
 		PROFILE="-v -xpg -g -xO4 -fast -native -fsimple -xcg92 -Dsuncc" ;;
 	*)
 		DEBUG="-g"
-		CFLAGS="-O"
+		CFLAGS=""
 		PROFILE="-g -p" ;;
         esac
 else
@@ -162,9 +162,9 @@ else
 		CFLAGS="-DDARWIN -fno-common -force_cpusubtype_ALL -Wall -g -O4 -ffast-math -fsigned-char"
 		PROFILE="-DDARWIN -fno-common -force_cpusubtype_ALL -Wall -g -pg -O4 -ffast-math -fsigned-char";;
 	*)
-		DEBUG="-g -Wall -W -D__NO_MATH_INLINES -fsigned-char"
-		CFLAGS="-O20 -D__NO_MATH_INLINES -fsigned-char"
-		PROFILE="-O20 -g -pg -D__NO_MATH_INLINES -fsigned-char" ;;
+		DEBUG="-g -Wall -W -fsigned-char"
+		CFLAGS="-fsigned-char"
+		PROFILE="-g -pg -fsigned-char" ;;
         esac
 fi
 CFLAGS="$CFLAGS $cflags_save"
